### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by wandji20**
In this milestone, We have implemented the styling of the different components based on the zeplin design.

### CODE REVIEW

**Comment by mimipeshy** [comment]
## STATUS : CHANGES REQUESTED 🔴 ♻️ 
Hello @wandji20 and @SamArsanios 👋🏾 👋🏾 congratulations on making it this far 😄 . 

#### Highlights

- Your UI looks almost identical to the given design✔️ 
- No linter errors are forming ✔️ 
- I can add a new book with no errors ✔️ 

#### To improve
Let's work on the following areas ⬇️ 
- [x] The Filter category should be in the navbar, kindly place it there
image
- [x] The live link demo should contain a live version of the styled app. Kindly work on this as well
- [x] Check out inline comments as well


That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

File: src/App.css [hunk 1]
```diff
@@ -36,4 +35,4 @@
   to {
     transform: rotate(360deg);
   }
-}
+} */
```
Comment (mimipeshy) [inline src/App.css:38]: - [ ] Lets not submit commented out code which makes the codebase to look untidy. Fix this as well

Comment (mimipeshy) [inline src/App.css]: - [x] Lets not submit commented out code which makes our codebase untidy. Kindly fix this too

**Review by enaburekhan — APPROVED** [review]
# Status: Approve:heavy_check_mark:
**Hi team!
Congratulations:clap::clap:. Thank you for implementing all the changes requested by the last reviewer.
Move on with your next project:walking:.**

Do have questions/comments? kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan.

**Pull request merged**